var/spack/repos/builtin/packages/apex/package.py

```diff
@@ -115,6 +115,9 @@ class Apex(CMakePackage):
     # Conflicts
     conflicts("+jemalloc", when="+gperftools")
     conflicts("+plugins", when="~activeharmony")
+    # Compatibility fixed in 2.6.0 with
+    # https://github.com/UO-OACISS/apex/commit/4a7bdbb93367c3b1172ccb978825c67316f8bf4a
+    conflicts("^otf2@3:", when="@:2.5.1")
```

Comment:
```suggestion
    conflicts("^otf2@3:", when="@:2.5")
```

Reply:
It's to cover the possibility of a 2.5.2 patch release which is unlikely to add support for otf2@3.